src/models/user.models.js

```diff
@@ -0,0 +1,69 @@
+import mongoose from 'mongoose';
+
+// User's schema
+
+const userSchema = new mongoose.Schema({
+    
+    email: {
+        type: String,
+        required: true,
+        trim: true,
+        match: /.*@.*\..*/
+        // validate: [isEmail, 'invalid email'],
+        // createIndexes: { unique: true }
+    },
+    password: {
+        type: String,
+        required: true
+    },
+    name: {
+        type: String,
+        required: true,
+        trim: true,
+        minLenght: 3,
+        maxLenght: 35
+    },
+    surname: {
+        type: String,
+        required: true,
+        trim: true,
+        minLenght: 3,
+        maxLenght: 35
+    },
+    birthDate: {
+        type: Date,
+        required: true,
+        trim: true
+    },
+    gender: {
+        type: String,
+        enum: ['Hombre', 'Mujer', 'Prefiero no decir'],
+        default: 'Prefiero no decir',
+        // required: true
+    },
+    avatar: {
+        type: String,
+        trim: true
+    },
+    company: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "ranking",
+        required: true
+    },
+    myEvents: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "events",
+        required: true
+    },
```

Comment:
@vcosmedev Deberia ser un arreglo donde se encontren los eventos
```suggestion
    events: [{
        type: mongoose.Schema.Types.ObjectId,
        ref: "events"
    }],
```